.github/runner/deployment.yaml.tmpl

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: github-runner-{{getenv "GITHUB_RUN_ID"}}
+  namespace: github-runner
+  labels:
+    app: github-runner
+    pod: github-runner-{{getenv "GITHUB_RUN_ID"}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: github-runner
+      pod: github-runner-{{getenv "GITHUB_RUN_ID"}}
+  template:
+    metadata:
+      labels:
+        app: github-runner
+        pod: github-runner-{{getenv "GITHUB_RUN_ID"}}
+    spec:
+      priorityClassName: high-priority
+      automountServiceAccountToken: false
+      containers:
+      - name: github-runner
+        image: us.gcr.io/rasa-platform/github-runner:2.262.1
```

Comment:
do we need to maintain that image somehow? Are the steps to create the image documented?

Reply:
Yes, we have a repository where we can build a new image. Of course, this step and way more will be described in docs related to the model regression tests :)